docs/data-sources/vmware_cloud_director_organization.md

```diff
@@ -0,0 +1,77 @@
+---
+subcategory: "VMware Cloud Director (VCD)"
+---
+
+# ovh_vmware_cloud_director_organization (Data Source)
+
+Get VMware Cloud Director organization details
+
+## Example Usage
+
+```terraform
+data "ovh_vmware_cloud_director_organization" "organization" {
+  organization_id = "<VCD organization ID>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `organization_id` (String) Organization ID
+
+### Read-Only
+
+- `current_state` (Attributes) Current status of VMware Cloud Director organization service (see [below for nested schema](#nestedatt--current_state))
```

Comment:
```suggestion
- `current_state` (Attributes) Current state of VMware Cloud Director organization service (see [below for nested schema](#nestedatt--current_state))
```

I guess since the name?